colossalai/shardformer/policies/gpt2.py

```diff
@@ -49,6 +50,16 @@ def module_policy(self):
                     target_module=col_nn.DropoutForParallelInput,
                 ),
             ])
+            if self.shard_config.enable_sequence_parallelism:
+                policy[GPT2Model] = ModulePolicyDescription(
```

Comment:
What if user enable TP and SP Simultaneously? It seems you overwrite the GPT2Model's policy. Shall we modify it instead of overwrite?

Reply:
change to modify type